### PR TITLE
Add deepin Intel-SIG

### PR DESCRIPTION
**Pull request by quanxianwang**
Intel will take part in deepin OS community as a whole. 

- We will focus on Intel platform enabling and related Intel features including AI, dGPU, oneAPI, openVino if possible.
- Intel will continually release new platforms and keep sync with upstream and deepin OS based on customer request.
- if possible, Intel will cooperate with deepin community to support OEMs including Dell, HP, Lenovo and more to support Intel platforms.

### CODE REVIEW

**Comment by quanxianwang** [comment]
多加了一个成员，这样能够按照deepin SIG的策略，超过两人。后续会根据需要加入英特尔相关的专家。多谢！

File: sig/deepin-Intel/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+team: sig-deepin-Intel
+repos:
+  maintained:
+    - sig-deepin-Intel
+  package:
+members:
+  - handle: quanxianwang
+    id:803339
+  - handle: Baolizha
+    id:29588229
```
Comment (BLumia) [inline sig/deepin-Intel/metadata.yml]:
```suggestion
    id: 803339
  - handle: Baolizha
    id: 29588229
```

File: sig/deepin-Intel/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+### 加入标准：
+
+1. 熟悉 Intel platform
+2. 了解开源协议
+2. 熟悉 debian 打包规则
+
+### 加入方法：
+
+1. 在[sig-deepin-Intel](https://github.com/deepin-community/sig-deepin-Intel/issues)提交issues 说明想加入的原因
+2. 订阅邮件列表 [deepin-Intel](https://www.freelists.org/list/deepin-Intel)
```
Comment (BLumia) [inline sig/deepin-Intel/README.md:28]: 这个列表目前不存在，是打算后续创建吗？如果不打算建这个邮件列表的话，实际可以怎样联系到并加入就可以留怎样的方式。我们并不强制使用 freelists 列表的。

File: sig/deepin-Intel/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Deepin Intel 小组
```
Comment (BLumia) [inline sig/deepin-Intel/README.md]:
```suggestion
# deepin Intel 小组 
```

deepin 作为品牌名称的时候，首字母是小写的。

File: sig/deepin-Intel/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Intel will continually release new platform and keep sync with upstream and deepin OS based on customer request.
+Also Intel will bring more features into deepin OS such as openVino, oneAPI, Intel GPUs if possible.
+Intel will cooperate with deepin community to support OEMs including Dell, HP, Lenovo and more to support Intel platforms.
+
+负责Intel platform的维护和更新工作。
+
+## 活动范围与目标
+
+目标：对Intel platform的维护和更新，跟进上游版本以及BUG修复，参与Intel platform上游社区建设等工作,保证维护的及时、安全、可靠。
```
Comment (BLumia) [inline sig/deepin-Intel/README.md]:
```suggestion
目标：对Intel platform的维护和更新，跟进上游版本以及BUG修复，参与Intel platform上游社区建设等工作，保证维护的及时、安全、可靠。
```

File: sig/deepin-Intel/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+活动范围: [社区论坛](https://bbs.deepin.org/)[deepin Community](https://github.com/deepin-community/) [GitHub issue](https://github.com/linuxdeepin/developer-center/issues)[deepin-Intel](https://github.com/deepin-community/sig-deepin-Intel)
```
Comment (BLumia) [inline sig/deepin-Intel/README.md]:
```suggestion
活动范围: [社区论坛](https://bbs.deepin.org/)[deepin Community](https://github.com/deepin-community/) [GitHub issue](https://github.com/linuxdeepin/developer-center/issues) [deepin-Intel](https://github.com/deepin-community/sig-deepin-Intel)
```

另外如果小组自身的主要活动范围只有最后写的 sig-deepin-Intel 仓库的话，只写最后一个也可以。如果确实会在社区 bbs 活动以及关注 deepin-community 组织下的所有相关动态的话，可以留着前面相应的两个链接。

File: sig/deepin-Intel/README.md (hunk 2 above)
Comment (quanxianwang) [reply sig/deepin-Intel/README.md:28]: 我已经在申请 freelist

File: sig/deepin-Intel/metadata.yml (hunk 1 above)
Comment (quanxianwang) [reply sig/deepin-Intel/metadata.yml]: 多谢，已经改了，重新提交。多加一个空格。

File: sig/deepin-Intel/README.md (hunk 3 above)
Comment (quanxianwang) [reply sig/deepin-Intel/README.md]: 改了。后续提交的commit。多谢

File: sig/deepin-Intel/README.md (hunk 4 above)
Comment (quanxianwang) [reply sig/deepin-Intel/README.md]: 修改完毕，后续的commit，多谢

File: sig/deepin-Intel/README.md (hunk 5 above)
Comment (quanxianwang) [reply sig/deepin-Intel/README.md]: 修改完毕。留着社区活动链接把，有可能英特尔会参与

**Comment by quanxianwang** [comment]
多谢。都已修改。

**Comment by quanxianwang** [comment]
README的deepin小写，然后metadata的deepin也应该小写。两个都改了过来。

**Comment by quanxianwang** [comment]
freelist is fine now. This is the email.

[deepin-intel@freelists.org](mailto:deepin-intel@freelists.org).

Welcome to the FreeLists community!

Your FreeLists mailing list is ready for use. Please read this document carefully and keep it for future reference.

All functions of your list can be controlled through the web by logging in at <https://www.freelists.org/login.html> To take full advantage of all FreeLists features, please login and take a look around. Many special features that may appeal to your subscribers are not turned on by default. 
Please send a test message to your list to make sure things are functioning correctly.

Some important starter info:

- To post on your mailing list, simply send email to [deepin-intel@freelists.org](mailto:deepin-intel@freelists.org). 

- Subscribers can join your list by sending email to [deepin-intel-request@freelists.org](mailto:deepin-intel-request@freelists.org) with 'subscribe' in the Subject field OR by visiting your list page at <https://www.freelists.org/list/deepin-intel>.

**Review by BLumia — APPROVED** [review]

**Review by RevySR — APPROVED** [review]

**Review by shiptux — APPROVED** [review]

**Review by zccrs — APPROVED** [review]

**Pull request merged**